java/component/keyple-core/src/main/java/org/eclipse/keyple/core/seproxy/plugin/local/AbstractObservableLocalReader.java

```diff
@@ -140,6 +152,130 @@ public AbstractObservableLocalReader(String pluginName, String readerName) {
         super(pluginName, readerName);
     }
 
+    /**
+     * Add a {@link ObservableReader.ReaderObserver}.
+     * <p>
+     * The observer will receive all the events produced by this reader (se insertion, removal,
+     * etc.)
+     *
+     * @param observer the observer object
+     */
+    @Override
+    public final void addObserver(ObservableReader.ReaderObserver observer) {
```

Comment:
in addObserver(ObservableReader.ReaderObserver observer), the parameter isn't defined as final